lib/contacts/publish_finder.rb

```diff
@@ -0,0 +1,8 @@
+module Contacts
+  class PublishFinder
+    def self.publish(presenter)
+      presented = presenter.present
+      Contacts.publishing_api.put_content_item(presented[:base_path], presented)
+    end
+  end
+end
```

Comment:
This class is pretty much identical to `Contacts::RegisterContact`.  Can we unify these 2 please.